### DISCUSSION TITLE
Customise the set picker dropdown with icons

### DISCUSSION THREAD

**Discussion by simonridley** [statamic/cms#4451] [General]
Would be nice to have an icon for each set as the naming can be a bit ambiguous for the user

If anyone has done this would you be interested in sharing, or create an addon perhaps?

![image](https://user-images.githubusercontent.com/5379986/136960417-a3237c36-fca1-4060-8a4e-e23e0f98b821.png)

**Comment by jasonvarga** [comment]
See https://github.com/statamic/ideas/issues/361
